helloworld.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
```

Comment:
Just a comment on this file in general - do we even need it? Running the main program with no arguments ```python3 slowmovie.py``` is basically the same thing. It will load the test move from the default Videos directory and play it. I had issues getting this to run when cloning this PR and it got me wondering why duplicate all this code for a test case?

Reply:
That's a good point, maybe it's a relic of the original development process... Including the test.mp4 file kind of fulfills the same purpose

Reply:
If it feels like we should take action on this, would we delete the helloworld file and change instructions on [line 39 of README.md](https://github.com/TomWhitwell/SlowMovie/pull/29/files/06b1963d2c2a62dd16195252e6a886b7ce67f42e#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R39)?

Reply:
That was my thought, change the test instructions to just try running the program without args and load that test file. Just looking at it now it has a lot of pieces that were modified in the main program, it's already quite out of date and will only get worse.

Reply:
Implemented in 218a84f. We'll have to make sure Tom's medium post update includes this as it causes a big change in the instructions.

Reply:
The idea for helloworld.py was to have a very minimal program that users who wanted to learn or develop could look at to get a sense how to call the epaper, extract images from file, dither those images, without all the overhead of config etc.
Maybe just move helloworld to the extras folder?

Reply:
Sounds good, Tom. I'll add it back in and edit it to match `slowmovie.py` a bit more.

Reply:
Thanks for the input, @TomWhitwell, I see now that it's important to have a simpler case that people can learn from. I added it back in the Extras folder via e1e3635